### PR TITLE
Replace Ruby Embulk::EmbulkRunner into Java org.embulk.EmbulkRunner.

### PR DESCRIPTION
**Pull request by dmikurube**
@muga Finally! We don't need `*.rb` files during the bootstrap now. (Several `runScriptlet` is still required, though.)

Can you have a look?

### CODE REVIEW

**Comment by dmikurube** [comment]
@muga Updated: some options (e.g. `-o`) were not working correctly.

**Comment by dmikurube** [comment]
@muga Updated the logging from `EmbulkRunner` to use slf4j.

**Comment by dmikurube** [comment]
jfyi, I'd release v0.8.28 when this PR is merged. Please ping me if you have concerns.

**Comment by muga** [comment]
I started taking a look.

File: embulk-cli/src/main/java/org/embulk/cli/EmbulkRun.java [hunk 1]
```diff
@@ -13,6 +13,8 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import org.embulk.EmbulkEmbed;
```
Comment (dmikurube) [inline embulk-cli/src/main/java/org/embulk/cli/EmbulkRun.java]: Okay, will remove.

**Comment by dmikurube** [comment]
@muga Thanks for taking a look.

The main reason why I located `EmbulkRunner` in `embulk-core` was that I just mechanically converted `lib/embulk/command/*` into `embulk-cli`, and `lib/embulk/*` into `embulk-core`. `EmbulkRunner` is a conversion from `lib/embulk/runner.rb`.

Another reason was that `EmbulkRunner` was Ruby `Embulk::Runner` which looked like a kind of "core" object throughout Embulk.

It's okay to move it to `embulk-cli` now, but I think we will anyways optimize and restructure things later gradually. Since these are big changes, I believe simple mechanical conversion works fine. Optimizing would be done in later changes seeing actual usages.

To make it clear, I've put the comment below there.
```
 * NOTE: Developers should not depend on this EmbulkRunner class. This class is created tentatively, and may be
 * re-implemented again in a different style.
```

File: embulk-cli/src/main/java/org/embulk/cli/EmbulkRun.java (hunk 1 above)
Comment (dmikurube) [inline embulk-cli/src/main/java/org/embulk/cli/EmbulkRun.java]: Removed in 92976c6.

**Comment by muga** [comment]
@dmikurube Thank you for explaining. I'm also OK to keep them in embulk-core now. Will refactor and restructure them guradually.

**Comment by dmikurube** [comment]
Thanks! Merging, and releasing v0.8.28.

**Pull request merged**